library/build.gradle

```diff
@@ -44,19 +44,18 @@ android {
 }
 
 dependencies {
-    compile 'com.loopj.android:android-async-http:1.4.9'
-    compile 'org.jboss.aerogear:aerogear-android-push:4.1.0'
-
     testCompile 'junit:junit:4.12'
-
+    compile 'com.squareup.okhttp3:okhttp:3.8.1'
+    compile 'com.squareup.okhttp3:logging-interceptor:3.8.1'
     androidTestCompile 'com.android.support:support-annotations:25.3.0'
     androidTestCompile 'com.android.support.test:runner:0.5'
     androidTestCompile 'com.android.support.test:rules:0.5'
-
     androidTestCompile 'org.mockito:mockito-core:1.9.5'
     androidTestCompile 'com.google.dexmaker:dexmaker:1.1'
     androidTestCompile 'com.google.dexmaker:dexmaker-mockito:1.1'
     androidTestCompile 'com.squareup.okhttp:mockwebserver:2.4.0'
+    compile 'com.squareup.okhttp:okhttp:2.4.0'
+    compile 'com.android.support:support-annotations:25.3.0'
```

Comment:
why 2 different versions of the `okhttp` modules?

Reply:
I even didn't know how this appeared here. One of them is not used.

Reply:
One is used in unit tests and one in application. They are different libraries. 
I used recent version so mockwebserver versions may need to be adjusted when fixing unit tests at later stage.

Reply:
replace `compile` with `implementation` ?